lib/SimplePie/SimplePie/Cache/File.php

```diff
@@ -117,7 +117,7 @@ public function save($data)
 	/**
 	 * Retrieve the data saved to the cache
 	 *
-	 * @return array Data for SimplePie::$data
+	 * @return array|false Data for SimplePie::$data
```

Comment:
Please do not change code in SimplePie. I have another pending task to reduce the diff with upstream